.github/workflows/ci.yml

```diff
@@ -271,7 +271,7 @@ jobs:
         run: curl -LsSf "https://github.com/astral-sh/uv/releases/latest/download/uv-aarch64-apple-darwin.tar.gz" | tar -xvz
 
       - name: "Check cache compatibility"
-        run: python scripts/check_cache_compat.py --uv-current ./uv --uv-previous ./uv-aarch64-apple-darwin/uv
+        run: python3.8 scripts/check_cache_compat.py --uv-current ./uv --uv-previous ./uv-aarch64-apple-darwin/uv
```

Comment:
No real reason for this test  test to use brew's Python. We may want to change that.

Reply:
I think it’s intentional to have a test for Brew Pythons though.

Reply:
Yeah but we don't need it in the cache compatibility test, just the system check.